tests/unit/test_bids_utils.py

```diff
@@ -81,3 +86,66 @@ def test_bidsmap_scans_run_plus():
     scans = [("1", "run+"), ("2", "run+"), ("3", "run+")]
 
     assert bidsmap_scans(scans) == [("1", "run-01"), ("2", "run-02"), ("3", "run-03")]
+
+
+def test_bidsify_dicom_headers(mocker):
```

Comment:
I'll need a short explanation on this one. I don't get how the mock works 🤪

Reply:
Fair, mocks are weird. I'll do it below